pints/__init__.py

```diff
@@ -140,6 +140,7 @@ def version(formatted=False):
     SumOfErrors,
     MeanSquaredError,
     RootMeanSquaredError,
+    NormalisedRootMeanSquaredError,
```

Comment:
Same here

Reply:
But SumofErrors is before MeanSquaredError... Which bit is alphabetically ordered?

Reply:
They should have been!
You can fix all of them :D